barely_json/__init__.py

```diff
@@ -138,8 +136,8 @@ def decode_match(match):
           null | true | false | illegal)
 
 
-def default_resolver(value):
-    '''
+def default_resolver(value_):
```

Comment:
I understand what you're trying to achieve, but IMO the better approach would be to move the grammar into a separate module so that it doesn't pollute the main module's namespace.